core/templates/dev/head/css/oppia.css

```diff
@@ -2646,6 +2646,36 @@ md-card.preview-conversation-skin-supplemental-card {
   margin: 0;
   padding: 0;
 }
+/* Editor Tutorial Welcome Modal */
+.oppia-welcome-modal .modal-dialog {
+  position: fixed;
```

Comment:
Please alphabetize properties within each CSS rule -- this makes them easier to find in the future.